http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20230911_create_app_usage_table.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="qi" id="qi_create_app_usage_table">
+        <createTable tableName="APP_USAGE">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="appId" type="BIGINT(20)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="startTime" type="TIMESTAMP(6)" defaultValue="0000-00-00 00:00:00.000000">
+                <constraints nullable="false"/>
+            </column>
+            <column name="stopTime" type="TIMESTAMP(6)" defaultValue="1970-01-01 00:00:01.000000">
```

Comment:
(not having too much scala & Leo knowledge)Would that be easier if we make this nullable, then insert null when app is started.
I am wondering if that would make the potential metrics/dashboard less confusing for running apps... 
(was thinking the BQ can just be where `stopTime is not null` but using the current style won't make too complicated).

Reply:
if we have `stopTime` is nullable, then mysql will update other `TIMESTAMP` fields automatically anytime the row is updated. This means `startTime` will be updated when we update `stopTime`...the only way I found to prevent that is to provide a default value

Reply:
could you say more about that?  that is surprising behavior.